packages/flutter/lib/src/painting/image_cache.dart

```diff
@@ -101,26 +100,23 @@ class ImageCache {
 
   /// Evicts a single entry from the cache, returning true if successful.
   ///
-  /// [key] is usually an image's corresponding [ImageProvider]. For example,
-  /// if you have an image in you Flutter app like the following.
-  ///
-  /// ```dart
-  ///     Widget build(BuildContext context) {
-  ///       return new Image.network(url);
-  ///     }
-  /// ```
+  /// [key] is usually an image's corresponding [ImageProvider]. To remove an
```

Comment:
nit: `The [key] argument is...` lets you avoid starting the sentence with a lowercase letter.